src/js/yeldaWebchatInjector.js

```diff
@@ -264,14 +275,33 @@ class YeldaChat {
   messageListener (event) {
     if (event.data === 'closeChat' || event.message === 'closeChat') {
       this.closeChat()
-    } else if (event.data === 'openChat' || event.message === 'openChat') {
+      return
+    }
+    
+    if (event.data === 'openChat' || event.message === 'openChat') {
       this.openChat()
-    } else if (event.data && event.data.event && event.data.event === 'openLightGallery') {
+      return
+    }
+    
+    if (event.data && event.data.event && event.data.event === 'openLightGallery') {
       if (!event.data.mediaSources || !event.data.mediaSources.length) {
         return
       }
 
       this.handleLightGallery(event.data)
+      return
+    }
+
+    /**
+     * Custom event to send to Yelda to keep track of the webchat ability to receive a new message
+     * If isSendingMessage data is true, it means that a user message has already been sent to the webchat and 
+     * is still waiting for an answer
+     * If isSendingMessage data is false, the webchat is ready to receive new user messages
+     */
+    if (event.data && event.data.event && event.data.event === 'isSendingMessage') {
+      if (event.data.hasOwnProperty('data')) {
+        window.dispatchEvent(new CustomEvent('isSendingMessage', { detail: event.data.data }) )
```

Comment:
As you added return ( :+1:) to other events management case, could be great to keep consistency and do the same here :)